dt-core/admin/admin-enqueue-scripts.php

```diff
@@ -85,6 +85,8 @@ function dt_options_scripts() {
         wp_register_style( 'jquery-ui', 'https://cdnjs.cloudflare.com/ajax/libs/jqueryui/1.12.1/jquery-ui.css' );
         wp_enqueue_style( 'jquery-ui' );
 
+        dt_theme_enqueue_style( 'material-font-icons', 'dt-core/dependencies/mdi/css/materialdesignicons.min.css', array() );
```

Comment:
@kodinkat 
Can we use the cdn instead of including the whole library in the theme?
Like we do in enqueue scripts?
`wp_enqueue_style( 'material-font-icons', 'https://cdn.jsdelivr.net/npm/@mdi/font@6.6.96/css/materialdesignicons.min.css' );`

Reply:
@corsacca Initially tried CDN approach, but it did not seem to provide a valid jquery handle (due to CORS errors) to remote `materialdesignicons.min.css` file; in order to parse and extract font icon names.

Therefore, including materials library locally, resolved this issue.